plugin/storage/es/mappings/jaeger-service-7.json

```diff
@@ -1,10 +1,22 @@
 {
-  "index_patterns": "*jaeger-service-*",
+  "index_patterns": "{{ ESPrefix }}jaeger-service-*",
```

Comment:
The "elasticsearch 7.x" tests are failing because of this line. `"*{{ ESPrefix }}jaeger-service-*"` should fix it.

Reply:
Fixed the tests